src/cpu/cpu.cpp

```diff
@@ -2435,9 +2435,15 @@ void CPU_ShutDown([[maybe_unused]] Section* sec) {
 	delete test;
 }
 
-void CPU_Init(Section* sec) {
+void CPU_Init(Section* sec)
+{
+	assert(sec);
+
 	test = new (std::nothrow) CPU(sec);
-	sec->AddDestroyFunction(&CPU_ShutDown,true);
+
+	constexpr auto changeable_at_runtime = true;
+	sec->AddDestroyFunction(&CPU_ShutDown, changeable_at_runtime);
 }
```

Comment:
For anyone reading the code and investigating some issue, seeing literal bools sprinkled about the code, like:

 - `AddFunction(func, true)`
 - `AddFunction(func, false)`

You either device to to read past them with the uneasy feeling of, *"well, I guess I don't know what happening with these functions ..."*, or you break your concentration and go lookupt hose calls just to double check (and yeah.. I can't remember how many times I had to grep and look them up!)

All of that to say, awesome full-pass cleanup commit, @johnnovak!

Reply:
One left in `src/dosbox.cpp` :-)

git grep 'Add.*Function' | grep 'true)\|false)'

```c++
secprop->AddInitFunction(&DOS_KeyboardLayout_Init, true);
```

Reply:
Ah, one slipped through somehow! 😄

Thanks for flagging it. I think we'll need to do this pattern for a long time, or maybe forever, because it doesn't look like named arguments are coming to C++ anytime soon (or ever, because it's hard to implement them in a backward compatible manner).